Scheduler/observing_blocks_maintel/AOS/BLOCK-T81.json

```diff
@@ -0,0 +1,594 @@
+                "rot_type": "PhysicalSky"
+            }
+        },  
+        {
+            "name": "maintel/take_triplet_comcam.py",
+            "standard": true,
+            "parameters": {
+                "exposure_time": "$exposure_time",
+                "dz": 1500,
+                "n_triplets": 1,
+                "program": "$program"
+            }
+        },
+        {
+            "name": "maintel/track_target.py",
+            "standard": true,
+            "parameters": {
+                "slew_icrs": {
+                    "ra": "$ra", 
+                    "dec": "$dec"
+                },
+                "rot_value": -39.6,
+                "rot_type": "PhysicalSky"
+            }
+        },  
+        {
+            "name": "maintel/take_triplet_comcam.py",
+            "standard": true,
+            "parameters": {
+                "exposure_time": "$exposure_time",
+                "dz": 1500,
+                "n_triplets": 1,
+                "program": "$program"
+            }
+        },
+        {
+            "name": "maintel/track_target.py",
+            "standard": true,
+            "parameters": {
+                "slew_icrs": {
+                    "ra": "$ra", 
+                    "dec": "$dec"
+                },
+                "rot_value": -32.4,
+                "rot_type": "PhysicalSky"
+            }
+        },  
+        {
+            "name": "maintel/take_triplet_comcam.py",
+            "standard": true,
+            "parameters": {
+                "exposure_time": "$exposure_time",
+                "dz": 1500,
+                "n_triplets": 1,
+                "program": "$program"
+            }
+        },
+        {
+            "name": "maintel/track_target.py",
+            "standard": true,
+            "parameters": {
+                "slew_icrs": {
+                    "ra": "$ra", 
+                    "dec": "$dec"
+                },
+                "rot_value": -25.2,
+                "rot_type": "PhysicalSky"
+            }
+        },  
+        {
+            "name": "maintel/take_triplet_comcam.py",
+            "standard": true,
+            "parameters": {
+                "exposure_time": "$exposure_time",
+                "dz": 1500,
+                "n_triplets": 1,
+                "program": "$program"
+            }
+        },
+        {
+            "name": "maintel/track_target.py",
+            "standard": true,
+            "parameters": {
+                "slew_icrs": {
+                    "ra": "$ra", 
+                    "dec": "$dec"
+                },
+                "rot_value": -18.0,
+                "rot_type": "PhysicalSky"
+            }
+        },  
+        {
+            "name": "maintel/take_triplet_comcam.py",
+            "standard": true,
+            "parameters": {
+                "exposure_time": "$exposure_time",
+                "dz": 1500,
+                "n_triplets": 1,
+                "program": "$program"
```

Comment:
I would add "reason": the ticket number for Reference wavefront analysis, and "note" : "reference_wavefront_generation" or something like that